lib/data/wrapper.js

```diff
@@ -21,6 +22,14 @@ if (config.backends.data === 'mem') {
 } else if (config.backends.data === 'multiple') {
     client = multipleBackendGateway;
     implName = 'multipleBackends';
+} else if (config.backends.data === 'cdmi') {
```

Comment:
This is the "legacy" way of setting a data backend. Is there any plan to have cdmi data backend working alongside other data backends? If so, we should have cdmi running under multiple but put a cdmi config in the loctionConstraintConfig. Let me know if we should discuss.